.travis.yml

```diff
@@ -150,16 +149,6 @@ install:
           export QT_API=pyqt;
       fi
 
-    # Reinstalling the required numpy version as it may be downgraded.
-    # TODO It may be removed when/if Sherpa becomes np version agnostic
-    # (maybe as soon as v4.8). See discussion in
-    # https://travis-ci.org/gammapy/gammapy/jobs/99867950#L639
-    - if $SHERPA; then
-          conda install --yes --channel astropy numpy=$NUMPY_VERSION iminuit;
-          conda install --yes --no-pin --channel https://conda.anaconda.org/cxc/channel/dev sherpa;
```

Comment:
If possible, I'd like to keep a build that uses the Sherpa dev version in addition.
Sherpa releases are ~ 1 year apart, and we want to keep testing against the dev version to make sure Gammapy keeps working.
Can you please add that build back into the matrix?

Also, I see that the Mac builds don't have scipy, Sherpa, .... all the optional dependencies installed and thus Gammapy isn't well-tested on Mac:
```
======= 175 passed, 193 skipped, 22 xfailed, 6 xpassed in 36.33 seconds ========
```
https://travis-ci.org/gammapy/gammapy/jobs/105696000#L673

Could you try installing all the optional dependencies like in the Linux Python 2.7 build where the coverage is measured for the Mac builds?
I think it'll just work.
Or I could make a separate PR with this change once this one is merged.